netaddr.go

```diff
@@ -659,6 +660,22 @@ func (ip IP) IsGlobalUnicast() bool {
 		!ip.IsLinkLocalUnicast()
 }
 
+// IsUnspecified reports whether ip is an unspecified address, either the IPv4
+// address "0.0.0.0" or the IPv6 address "::".
+//
+// Note that the ip zero value is not an unspecified address. Use IsZero to
+// check for the zero value instead.
+func (ip IP) IsUnspecified() bool {
+	if ip.Is4() {
```

Comment:
did you write it like this for performance or for clarity?

Because I'd think the one-liner would be the same speed & clarity:

    return ip == IPv4(0,0,0,0) || ip == IPv6Unspecified()

is it not?

In any case, I don't care much how it's written exactly. Just curious.

Reply:
Clarity and consistency with other methods. No preference here.

Reply:
Changing to simple return with ||.